### DISCUSSION TITLE
Which country(ies) have the list of pre-approved 👶🍼 names?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1044] [Galaxy Trivia] [answered]
**Bonus:** Decrypt the string *Brfxxccxxmnpcccclllmmnprxvclmnckssqlbb11116*

**Comment by peterbaccaro** [comment]
The following countries have a list of pre-approved names:

- **Iceland**: Iceland has a strict naming law and maintains a list of approved names that parents must choose from. If they wish to use a name not on the list, they must seek approval from the Icelandic Naming Committee.

- **Denmark**: Denmark also has a list of pre-approved names, and parents are required to choose from this list. If they want to use a name not on the list, they must seek permission from the government.

Brfxxccxxmnpcccclllmmnprxvclmnckssqlbb11116 is a name intended for a Swedish child who was born in 1991. Parents Elisabeth Hallin and Lasse Diding gave their child this name in protest of the Naming law. The couple claimed that the name was pronounced **Albin**, but it was clearly not in line with the accepted naming rules in Sweden. The name was rejected.

**Comment by Ben74x** [comment]
Countries with pre-approved baby names: 

Denmark: The country has a list of about 7,000 pre-approved names. Parents can choose a name outside of this list, but they have to get approval from the Danish government.

Iceland: With the country's strong emphasis on linguistic cohesion and preserving their cultural heritage, Iceland has a specific Naming Committee to approve or reject names that do not fit within their guidelines.

Germany: The German government has certain rules regarding names. The name must indicate the child's gender and cannot be potentially harmful or demeaning to the child. Unusual names may be rejected.

Norway: Norway also maintains a list of pre-approved names, with the aim to protect children from names that could be harmful or offensive.

Sweden: The country has a naming law to ensure that names will not cause discomfort for the person who uses it.

China: Names must be able to be translated into Mandarin and characters that are difficult to write or are extremely rare are typically rejected.

Portugal: Portugal has an 82-page official document that details which names are accepted and which are not.



Brfxxccxxmnpcccclllmmnprxvclmnckssqlbb11116" is actually a proposed name by a couple in Sweden, which was subsequently rejected by the Swedish naming laws

**Comment by ParanoidUser** [comment]
**Extra:** Post one of the rarest 2019 🇺🇸 baby names with an emoji.

**Reply by peterbaccaro** [reply]
Here are three of the rarest 2019 🇺🇸 baby names with an emoji:
1. SAILOR ⛵
2. BLUBELLE 🔵🔔
3. SEVEN 7️⃣

**Comment by thuongtruong1009** [comment, marked as answer]
Here are some countries known to have had restrictions or lists of pre-approved names:

1. **China**: China has a list of pre-approved names, and parents must choose a name from this list. The list is managed by the Chinese government to preserve cultural identity and avoid names with offensive or sensitive connotations.

2. **Iceland**: Iceland has an official register of approved names. If parents want to give their child a name not on the list, they need to seek approval from the Icelandic Naming Committee.

3. **Denmark**: Denmark has strict naming laws, and parents can choose names only from a pre-approved list of names. However, exceptions can be made if parents obtain permission from the church or local authorities.

4. **Germany**: Germany has certain naming restrictions, and names must comply with the country's naming law. Gender-neutral names and last names as first names are not allowed, and the name must indicate the gender of the child.

5. **Japan**: While Japan doesn't have an official pre-approved list of names, there are naming restrictions to prevent names that can cause issues or misunderstandings.

6. **Malaysia**: Malaysia has guidelines for naming babies, and the National Registration Department has a list of names that are considered acceptable for Muslim children.

7. **Mexico**: In Mexico, some states may have naming restrictions, and parents might need to follow specific guidelines when registering their child's name.

Remember that naming regulations can vary within countries, and some regions or states might have their own additional rules. Additionally, laws may have changed since my last update, so it's always best to consult the most recent information from official government sources to ensure compliance with the current naming regulations in a particular country.
